acceptance-tests/src/test-support/java/org/hyperledger/besu/tests/acceptance/dsl/node/BesuNode.java

```diff
@@ -462,6 +464,13 @@ JsonRpcConfiguration jsonRpcConfiguration() {
     }
   }
 
+  public String jsonRpcListenHost1() {
```

Comment:
If a node is 5-chain node, then are we going to have jsonRpcListenHost1()...jsonRpcListenHost5() and ...port1()...port5()? So, I think we can keep this for the current PR. I will do a refactor later, where may be some index like chainId may be used to query upon.

Reply:
This is just to differentiate this from the other jsonRpcListHost method which returned an optional parameter. I should have a better name for this method.

Reply:
Yup. I understood this, when I got ahead in the review